### PR TITLE
add s3gw for session-log subsystem

### PR DESCRIPTION
**Pull request by umezawatakeshi**
part of https://github.com/cybozu-go/neco/issues/1411

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: session-log/base/s3gw.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      labels:
+        app.kubernetes.io/name: s3gw
+    spec:
+      containers:
+        - name: s3gw
+          image: quay.io/cybozu/s3gw:1.0.0
+          args:
+          - --listen=:20080
+          - --use-path-style
+          - --hosts-allow=10.72.48.0/24 # bastion subnet for boot servers
```
Comment (ysksuzuki) [inline session-log/base/s3gw.yaml:22]: `--hosts-allow` doesn't need to be patched for bastion networks for tokyo-0 and osaka-0?  `10.72.48.0/24` is for stage-0.

Comment (umezawatakeshi) [reply session-log/base/s3gw.yaml:22]: Yes, it must be patched according to env.

**Comment by yokaze** [comment]
LGTM

**Review by yokaze — APPROVED** [review]
LGTM

**Pull request merged**